omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -401,6 +400,18 @@
             Name="datadog.yaml"
             Source="$(var.EtcFiles)\datadog.yaml.example"/>
       </Component>
+
+
+      <Component Id="install_info"
+                 Guid="260CEE46-E9E9-4EEF-A065-969524970B68"
+                 Permanent="no">
+        <Condition>
+          <![CDATA[UILevel > 2]]>
```

Comment:
Note: The [`UILevel` property](https://docs.microsoft.com/en-gb/windows/win32/msi/uilevel) is set to 2 for a completely silent installation and over 2 for installs that have some GUI. This means we create the install_info file even when the UI is not fully available.